krux_cloud_health/cli.py

```diff
@@ -0,0 +1,101 @@
+# -*- coding: utf-8 -*-
+#
+# © 2016 Krux Digital, Inc.
+#
+"""
+CLI tools for accessing Krux Cloud Health Tech
+"""
+
+#
+# Standard libraries
+#
+
+from __future__ import absolute_import
+
+#
+# Third party libraries
+#
+
+from datetime import date, timedelta
+
+#
+# Internal libraries
+#
+
+import krux.cli
+from krux.logging import get_logger
+from krux.cli import get_group
+from krux_cloud_health.cloud_health import CloudHealth, NAME, INTERVAL, add_cloud_health_cli_arguments, get_cloud_health
+
+class Application(krux.cli.Application):
+    def __init__(self, name=NAME):
+
+        # Call to the superclass to bootstrap.
+        super(Application, self).__init__(name=name)
+
+        self.logger = get_logger(name)
+
+        try:
+            self.cloud_health = get_cloud_health(args=self.args, logger=self.logger, stats=self.stats)
+        except ValueError as e:
+            self.logger.error(e.message)
+            self.exit(1)
+
+        self.interval = self.args.interval
+
+        if self.args.set_date is not None:
+            self.date_input = self.args.set_date
+        else:
+            today = date.today()
+            if self.interval == 'daily':
+                self.date_input = '{}'.format(today - timedelta(days=1))
```

Comment:
Index-less format will not work in Python 2.6. Conver this to either `{0}` or `{date}` format.